kubernetes/pybot/base/deployment.yaml

```diff
@@ -0,0 +1,63 @@
+---
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: pybot
+spec:
+  replicas: 1
+  revisionHistoryLimit: 1
+  template:
+    spec:
+      containers:
+      - name: app
+        image: operationcode/pybot:latest
+        imagePullPolicy: Always
+        ports:
+        - containerPort: 8000
+        resources:
+          requests:
+            memory: 200Mi
+            cpu: 100m
+        readinessProbe:
+          httpGet:
+            path: /healthz
```

Comment:
Looks like the path should be `/health` and port 5000 based on the current target group: https://us-east-2.console.aws.amazon.com/ec2/v2/home?region=us-east-2#TargetGroup:targetGroupArn=arn:aws:elasticloadbalancing:us-east-2:633607774026:targetgroup/pybot-targets/d99ef26df1864cd6